src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/DependencyInfoResourceV2Feed.cs

```diff
@@ -77,12 +77,12 @@ public DependencyInfoResourceV2Feed(V2FeedParser feedParser, SourceRepository so
 
                 return results;
             }
-            catch (Exception ex)
+            catch (Exception)
```

Comment:
Can you isolate exception type thrown when package is not found? I think we want to keep generic error message for any other failure.